pkg/node/manager.go

```diff
@@ -208,6 +208,37 @@ func replaceNodeRoute(ip *net.IPNet) {
 	}
```

Comment:
Nit in commit: "This only makes sense when tunneling is enabled; when tunelling is disabled.."